### PR TITLE
Fix tox, and some little tweaks

### PR DESCRIPTION
**Pull request by erikkemperman**
Hi @MainRo,

I noticed that the master branch (without any recent changes) fails when running `tox`:
https://travis-ci.org/erikkemperman/dooble/builds/526281783

Looking into this, I found that this is due to the new `tatsu` release, v4.4, which omits items from the AST if the value is empty. But the tests were expecting the key to be present anyway, with value `None`.

I fixed that within this PR, and added a few minor cosmetic changes.

Alternatively, I suppose, the grammar could be changed to enforce the keys but make the values optional.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -93,17 +93,17 @@ The generated image looks like this:
...
+* Each text line represents either an observable, or an operator
 * Observables are defined with dash lines
 * If an Observable starts with a letter, this letter is considered as a label
 * Each character represent a time span
-* Character *|* indicates an observable completion
-* Character *\** indicates an observable error
-* Character *>* indicates an observable continutation
-* The *+* character is used to define anchors of higher order observables
-* operators start with a *[* and end with a *]*
+* Character ``*`` indicates an observable completion
```
Comment (MainRo) [inline README.rst]: original text was wrong, but this one also: the character for completion is | (pipe).

File: tox.ini [hunk 2]
```diff
@@ -1,14 +1,17 @@
 [tox]
-envlist = py3
+envlist = py36, py37
```
Comment (MainRo) [inline tox.ini]: I don't think this is desirable: I use tox only to ease running tests and packaging. So keeping just py3 allows to use the python version present on the system. On development systems, we use the default python3 installation, on CI, the version used depends on the travis environment, i.e. tested on both 3.6 and 3.7 with your changes.

File: .travis.yml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 language: python
+dist: xenial
 python:
-- 3.6
+  - "3.6"
+  - "3.7"
 install: pip install -U tox-travis
 script: tox
 deploy:
```
Comment (MainRo) [inline .travis.yml]: with this change, a deploy will be triggered on both python 3.6 and 3.7, so the second one will fail. We chould use a matrix, something like that:

https://github.com/PacktPublishing/Migrating-from-Python-2-to-Python-3/blob/master/chapter08/packt-calc/.travis.yml

So we could add tests on macos and windows at the same time.

File: tox.ini [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
 
+[travis]
```
Comment (MainRo) [inline tox.ini]: if we use a matrix in travis.yml, then I am not sure that we need this section.

**Comment by erikkemperman** [comment]
Thanks for review and merge!

**Comment by MainRo** [comment]
Thanks to you for the fix, and now tests are running on more platforms.

**Pull request merged**